syntaxes/kubernetes.output.tmLanguage

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
The Helm tmLanguage file is in JSON format rather than XML.  Would it be impractical to align the two?  (It's not a big deal, anyone who can maintain one can maintain the other, it just looks a bit odd to me. *grin*)

Reply:
Make sense. Json is more readable. Have changed it to json.